presto-elasticsearch/src/main/java/io/prestosql/elasticsearch/client/ElasticsearchClient.java

```diff
@@ -195,27 +198,33 @@ private static RestHighLevelClient createClient(ElasticsearchConfig config, Opti
     {
         RestClientBuilder builder = RestClient.builder(
                 new HttpHost(config.getHost(), config.getPort(), config.isTlsEnabled() ? "https" : "http"))
-                .setRequestConfigCallback(
-                        configBuilder -> configBuilder
-                                .setConnectTimeout(toIntExact(config.getConnectTimeout().toMillis()))
-                                .setSocketTimeout(toIntExact(config.getRequestTimeout().toMillis())))
                 .setMaxRetryTimeoutMillis(toIntExact(config.getMaxRetryTime().toMillis()));
 
         builder.setHttpClientConfigCallback(clientBuilder -> {
```

Comment:
Rename `clientBuilder` to `ignored` and replace `httpClientBuilder` below with `clientBuilder`